app/models/join.rb

```diff
@@ -0,0 +1,7 @@
+class Join < ApplicationRecord
+  belongs_to :user
+  belongs_to :recruit
```

Comment:
http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#module-ActiveRecord::Associations::ClassMethods-label-Setting+Inverses

```suggestion
  belongs_to :user, inverse_of: joins
  belongs_to :recruit, inverse_of: joins
```